lib/seek/renderers/blob_renderer.rb

```diff
@@ -26,6 +26,10 @@ def can_render?
         fail 'needs to be implemented'
       end
 
+      def is_remote?
+        false
+      end
```

Comment:
I wonder if this needs a better name to explain what it is used for... maybe `external_embed?` or something

Reply:
Changed :)